cmd/brew-portable-package.rb

```diff
@@ -1,28 +0,0 @@
-# Build a portable Homebrew utility bottle.
-#
-# Usage: brew portable-package <formula> [formula...]
-
-odie "Need to specify at least one portable formula!" if ARGV.empty?
-
-if RUBY_VERSION.split(".").first.to_i < 2
-  safe_system "brew", "install", "ruby"
```

Comment:
FWIW, this removed script cannot be run on Tigerbrew yet either. Because `--skip-reloation` flag in `brew bottle` is not supported by it.